### ISSUE TITLE
[FP] "explode(...) limit can be used" when leaving out the first item in destructuring

### ISSUE THREAD

**Issue by jdreesen** [kalessil/phpinspectionsea#1463]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 7.2                       |

Current behaviour
--
When leaving out the first parameter of `list(...)` EA thinks the second one is the first and suggests adding the `$limit` parameter to `explode(...)` which changes the behavior of the code.
![image](https://user-images.githubusercontent.com/424602/70633022-22701480-1c30-11ea-92f7-e125b604ac16.png)

Expected behaviour
--
The inspection doesn't get triggered.

**Comment by rentalhost** [comment]
Confirmed.

```php
[ , $b ] = explode(',', 'a,b,c,d');  
// Should suggests: 
// explode(',', 'a,b,c,d', 1);
// But suggests: 
// explode(',', 'a,b,c,d', 2);
```

Additionally, it is not triggered when have more than one variable:

```php
[ $a, $b ] = explode(',', 'a,b,c,d');  
// Should suggests: 
// explode(',', 'a,b,c,d', 3);
```

**Comment by kalessil** [comment]
Will fix! Thank you for reporting @jdreesen and @rentalhost for confirming.

**Comment by kalessil** [comment]
Fixed!
